client/source/js/modules/model/cost-coverage.html

```diff
@@ -70,8 +67,8 @@
               </p>
               <p>
                 3. [Optional] Specify scale-up parameters:
-                <input type="number" name="scaleUpParameter" min="1" max="10" step="0.1" class="txbox __inline __xl" placeholder="e.g.{{defaultScaleUpParameter}}"  ng-model="scaleUpParameter" ng-change="updateCurves()">
-                <error-messages for="scaleUpParameter" rules="{min:1, max:10}"></error-messages>
```

Comment:
@robynstuart no< with 0, 0.8, etc. E.g. for OST with this set of parameters: ccparams: [0.9, 0.6, 400000, 0.8, 0]
ccplot: [] - I get absolutely flat line.